nucliadb/src/nucliadb/ingest/orm/entities.py

```diff
@@ -316,21 +311,18 @@ async def query_indexed_entities_group_names(node: AbstractIndexNode, shard_id:
             response: SearchResponse = await query_shard(node, shard_id, request)
             try:
                 facetresults = response.document.facets["/e"].facetresults
-                return {facet.tag.split("/")[-1] for facet in facetresults}
             except KeyError:
                 # No entities found
                 return set()
+            else:
+                return {facet.tag.split("/")[-1] for facet in facetresults}
```

Comment:
Is this intended? This changes means if the KeyError occurs in this line, it will be raised instead of ignored. Probably OK, just double-checking.

Reply:
Yes, if we don't find `/e` means there's no entities, but a `KeyError` here would mean indexing changed or is wrong, we should notice it